packages/workflow-core/src/lib/utils/definition-validator/extensions-validator.ts

```diff
@@ -21,67 +23,75 @@ export const extensionsValidator = (extensions: WorkflowExtensions, states: Stat
       pluginKind === 'email' ||
       pluginKind === 'kyc'
     ) {
-      validateApiPlugin(plugin as unknown as ISerializableHttpPluginParams, states)
+      validateApiPlugin(plugin as unknown as ISerializableHttpPluginParams, states);
```

Comment:
No change required - here is where a type guard of `isApiPlugin` or `isPlugin` would be used so type assertions won't be needed.